encyctng/static_src/sass/components/_related-media.scss

```diff
@@ -0,0 +1,128 @@
+@use 'config' as *;
+
+.related-media {
+    $root: &;
+    border-top: 1px solid var(--color--border);
+    margin-bottom: $spacer-small;
+
+    @include media-query(medium) {
+        margin-bottom: $spacer-medium;
+    }
+
+    &__heading {
+        color: var(--color--headings);
+        margin-bottom: 5px;
+        padding-bottom: 0;
+    }
+
+    &__sub-heading {
+        margin-bottom: $spacer-small;
+
+        @include media-query(medium) {
+            margin-bottom: $spacer-medium;
+        }
+    }
+
+    &__list {
+        display: flex;
+        flex-direction: column;
+        gap: $spacer-small;
+
+        @include media-query(medium) {
+            gap: $spacer-small-plus;
+            flex-direction: row;
+            flex-wrap: wrap;
+        }
+    }
+
+    &__item {
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        gap: $spacer-mini;
+
+        @include media-query(medium) {
+            flex-basis: calc(50% - #{$spacer-small-plus / 2});
+        }
+
+        @include media-query(large) {
+            flex-basis: calc(33.333% - #{$spacer-small-plus * 2 / 3});
+        }
+    }
+
+    &__image-wrapper {
+        position: relative;
+        width: 80px;
+        height: 80px;
+        flex-basis: 80px;
+        max-width: 80px;
```

Comment:
Do we need max and min widths here if we're explicitly setting width and height anyway?

Reply:
Good point, stuck in my supporting IE11 ways 👍